.gitignore

```diff
@@ -298,3 +298,9 @@ quickstart/csharp-unity/[Ll]ogs/
 # Linux executables
 quickstart/cpp-linux/helloworld
 samples/cpp/windows/console/samples/sample
+*.dll
+*.exe
```

Comment:
these are probably in the objd directory? so they should be excluded anyhow

Reply:
you mean they would be excluded even without me explicitly saying so?

Reply:
fixed gitignore as per wolfgang's instructions